src/sdk/models/jobs.ts

```diff
@@ -43,3 +43,49 @@ export interface TransferOptions {
 	source?: TransferPath;
 	destination?: TransferPath;
 }
+
+export interface Duration {
+	value?: number;
+	units?: string;
+}
+
+export interface JobExecutionStatistics {
+	bytes?: number;
+	files?: number;
+	files_pending?: number;
+	rate_limits?: number;
+	versions?: number;
+	versions_pending?: number;
```

Comment:
Add `[name: string]: number` to allow access to other stats not exposed here.

Reply:
That's a new one on me! https://devdocs.io/typescript/handbook/interfaces#optional-properties